### DISCUSSION TITLE
Reference to documentationUrl in message

### DISCUSSION THREAD

**Discussion by VG1UONW** [stoplightio/spectral#2498] [Q&A] [answered]
Hi!

Is there any way to reference documentationUrl in message property? 
For me it is important to see link to documentation in a console to address an issue.
I am specifying rules in yaml file and so far I see only this option:
```
...
  rule-name:
    message: |
      Some useful message
      https://google.com
    documentationUrl: https://google.com
...
```
Is there a fancy way to achieve this without duplicating url in message and documentaionUrl properties?

Thanks!

**Comment by P0lip** [comment, marked as answer]
Hey!
Unfortunately, that is not available at the moment, although we'd accept a feature request!
Do feel free to file an issue :)

**Reply by VG1UONW** [reply]
Thank you very much! I will make a feature request!
